test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Running Parity Full Test Sute
 
-FEATURES="--features json-tests"
+FEATURES="--features json-tests;ipc"
```

Comment:
Seems it might have been changed in beta, `stable` is complaining about missing feature on Travis.